### PR TITLE
Basic car class

### PR DESCRIPTION
**Pull request by Sergiy-1998**
Define a base class for auto.
Base class should contain:
Fields:
 - type (gas, electro, petrol)
 - model
 - year
 - max speed.
Method:
 - get car type

Define child classes for each type of car.
`get_car_type` method should return just an appropriate type of car.

Create a general class which should work in this way:

car = Car('gas', 'BMW', 2018, 280)
print(car.get_car_type())  # prints `Gas`
car.change_type('petrol')
print(car.get_car_type())  # prints `Petrol`.

### CODE REVIEW

**Comment by python-teacher** [comment]
Please change the name of this pull request.
It should be self-descriptive. It should be clear from the name what is this pull request about.

File: class_basic_car.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+car = Car('gas', 'BMW', 2018, 280)
+print(car.get_car_type()) # prints Gas
+car.change_type('petrol')
+print(car.get_car_type()) # prints Petrol.
+"""
+
+
+class Car:
+    def __init__(self, type, model, year, max_speed, ):
+        self.type = type
```
Comment (python-teacher) [inline class_basic_car.py]: Please try to avoid using built-in functions as a variable name.(It's according to PEP-8)

File: class_basic_car.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        return self.type
+
+    def change_type(self, types):
+        self.type = types
+
+
+car = Car('gas', 'BMW', 2018, 280)
+print(car.get_car_type())  # prints `Gas`
+car.change_type('petrol')
+print(car.get_car_type())  # prints `Petrol`.
```
Comment (python-teacher) [inline class_basic_car.py:41]: How it might print `Petrol`?

**Review by python-teacher — CHANGES_REQUESTED** [review]

File: class_basic_car.py (hunk 1 above)
Comment (Sergiy-1998) [reply class_basic_car.py]: done

File: class_basic_car.py (hunk 2 above)
Comment (Sergiy-1998) [reply class_basic_car.py:41]: the change_type function replaces the type on the specified one
and the get_car_type function outputs a new type

Comment (python-teacher) [reply class_basic_car.py:41]: Ok. After the latest changes, it prints as expected.

**Review by python-teacher — APPROVED** [review]

**Comment by python-teacher** [comment]
:+1:

**Pull request merged**